Finishing my Portfolio

bhskr44 opened the pull request:
For the first milestone in the process of creating your portfolio website, we need to set up a new repository and prepare it for development using best practices (e.g. liters). building all the sections of the mobile website using the template we choose a predetermined design given by Microverse.

bhskr44 committed:
Initial Commit

Added index.html and all other files

bhskr44 committed:
LInters Updated

New Linter File is added

bhskr44 committed:
Updated main.css

Indentation was corrected

bhskr44 committed:
@bhskr44 Updated main.css

Indentation was corrected

bhskr44 committed:
Update main.css

Indentation was corrected

bhskr44 committed:
Update main.css

Indentation was corrected

bhskr44 committed:
Update main.css

Indentation was corrected

bhskr44 committed:
Update README.md

Readme file Updated

bhskr44 committed:
Update README.md

new framework added

bhskr44 committed:
Updated main.css

Updated main.css

bhskr44 committed:
Update main.css

Updated main.css

bhskr44 committed:
Update main.css

Updated main.css

bhskr44 committed:
Update main.css

bhskr44 committed:
Update main.css

new line added to the main.css  no-missing-end-of-source-newline

bhskr44 committed:
Update main.css

bhskr44 committed:
Update main.css

bhskr44 committed:
Update main.css

indentation matched

bhskr44 committed:
Update main.css

Added new line before rule

bhskr44 committed:
Update main.css

bhskr44 committed:
Commit after the Changes requested

- li wrapped to ul
-  wrap paragraph into p
- icons in the a
-Removed bootstrap

bhskr44 committed:
Linters Adjusted

Linters Adjusted

bhskr44 committed:
Updated Main.css

Updated Main.css

bhskr44 committed:
Updated main.css

bhskr44 committed:
Figma File

Tried to match the figma file

bhskr44 committed:
Update main.css

Linter errors

bhskr44 committed:
Update main.css

bhskr44 committed:
Added New Section #myworks

Added new section My works

bhskr44 committed:
Fixing Linters

Fixing Linters

bhskr44 committed:
Project Updated with the contact us section

Project Updated with the contact us section

bhskr44 committed:
Updated According to linters

Updated According to linters

bhskr44 committed:
Updated Main.css

Updated Main.css

bhskr44 committed:
Update main.css

bhskr44 committed:
Update main.css

jerryowusu commented on README.md:
```diff
@@ -1,2 +1,51 @@
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
[OPTIONAL] You might consider taking this out of the readme file since it is just an instruction to help guide you. It does not need to be in the final version.

jerryowusu commented on css/main.css:
```diff
@@ -0,0 +1,356 @@
+  text-align: center;
+  margin-left: 20px;
+  margin-right: 20px;
+}
```
Kindly use CSS Grid for the main container that contains all of the individual work cards as stated in the project requirements below (check bullet point number 2)

![Screenshot from 2022-09-23 12-45-52](https://user-images.githubusercontent.com/65233860/191989296-bad51ba6-cdb9-49c6-9b99-4c322dfce903.png)

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,362 @@
+            hestiate to contact me.
+          </p>
+
+          <div class="social">LET’S CONNECT</div>
```
Kindly use an hx (h2-h6) tag for this one since it is a title/header instead of a div tag.

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,362 @@
+          <button type="button" class="get-resume-btn">Get My Resume</button>
+        </div>
+
+        <div class="accordation">
+          <div class="title">
+            <h1>Languages</h1>
+            <div class="icon">
+              <i class="fa fa-angle-down" aria-hidden="true"></i>
+            </div>
+          </div>
+          <div class="floating-elements">
+            <div class="single-elemnt">
+              <img src="image/language/js.png" />
+              <h2>Javascript</h2>
+            </div>
+          </div>
+          <div class="floating-elements">
+            <div class="single-elemnt">
+              <img src="image/language/html.webp" />
+              <h2>HTML</h2>
+            </div>
+          </div>
+          <div class="floating-elements">
+            <div class="single-elemnt">
+              <img src="image/language/css.webp" />
+              <h2>CSS</h2>
+            </div>
+          </div>
+        </div>
```
As stated in the project requirements, kindly use ul and li tags for the categories in `About Myself` section. Please make sure that the `Languages`, `Frameworks`, and `Skills` all use the ul and li tags. (check bullet point number 4)

![Screenshot from 2022-09-23 12-45-52](https://user-images.githubusercontent.com/65233860/191992447-7cf671f3-1141-4634-99c3-794f058385ed.png)

jerryowusu commented on css/main.css:
```diff
@@ -0,0 +1,356 @@
+  border-radius: 5px;
+}
+
+.seeproject > button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-color: #6070ff;
+  margin-top: 20px;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+}
+
+.get-resume > button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-color: #6070ff;
+  margin-top: 20px;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+}
+
```
Please kindly implement the active, pressed and hover effects of all the buttons. It is true that these effects are not seen on mobile versions but it is a must to implement them at this stage of the project :stuck_out_tongue_winking_eye:  You can check the desktop side of the figma design for the button effects. :pray:

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,362 @@
+              <li>
+                <a href="#"><i class="fa fa-facebook fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-twitter fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-instagram fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-github fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-linkedin fa-2x"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="get-resume">
+          <button type="button" class="get-resume-btn">Get My Resume</button>
+        </div>
+
+        <div class="accordation">
+          <div class="title">
+            <h1>Languages</h1>
+            <div class="icon">
+              <i class="fa fa-angle-down" aria-hidden="true"></i>
```
I am really impressed with your project so far. Keep it up. However, this sections feels cramped and not really conforming to the figma design. Kindly check the font-family, font-size, weight, padding and margins and make sure they are as close to the figma design as possible. 


Figma design
![Screenshot from 2022-09-23 15-24-33](https://user-images.githubusercontent.com/65233860/191996455-089cae23-3e1c-4ded-a093-04528be0ebeb.png)


Your design
![Screenshot from 2022-09-23 15-25-17](https://user-images.githubusercontent.com/65233860/191996570-ad20a4d1-d94d-4a18-a0d0-84cab04b4b94.png)

jerryowusu reviewed (changes requested):
Hi @bhskr44 :wink:, 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

![almost there](https://user-images.githubusercontent.com/65233860/191997774-53e45daf-cdd2-46af-a583-0a49251b59d4.jpg)


## Highlights
:heavy_check_mark: Good commit messages
:heavy_check_mark: Neat and well organized code
:heavy_check_mark: Excellent readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bhskr44 committed:
Main CSS and HTML updated according to the suggestion made during the Code Review

Main CSS and HTML updated according to the suggestion made during the Code Review

bhskr44 committed:
Updated Linters

Updated Linters

bhskr44 committed:
Update main.css

Update main.css

HAMMAS-SALEEM reviewed (approved):
# Status: Approved :green_circle:
![success](https://camo.githubusercontent.com/6ada59f6189497c1c2cae6d06f8154b7307a58643f232340e288140ed469eb28/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966)

## Hey @bhskr44 :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following the GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
